frontend/src/ui/components/CustomStatusPill/util.ts

```diff
@@ -51,12 +51,12 @@ export function buildStatusContext(
       }
       break;
     case DeploymentStatus.Preparing:
-      result.dotColor = '#ff9b04';
+      result.pillColor = '#ff9b04';
       result.deploymentStatusText = 'preparingInstance';
       result.tooltipText = 'preparingInstanceTooltip';
       break;
     case DeploymentStatus.ChangesPending:
```

Comment:
This sets the pill to "Publishing", which is right by the requirements. Is this correct, by logic?

Reply:
I think so because preparing is a part of publishing